src/main/resources/db/migration/V2__create_submitted_draft_adjudication_table.sql

```diff
@@ -0,0 +1,12 @@
+drop table if exists submitted_draft_adjudications;
+
+create table submitted_draft_adjudications
```

Comment:
Going to wait until we know how this is going to be accessed until any indexes are added.